### PR TITLE
Update aws-sources.md

### PR DESCRIPTION
**Pull request by bnartiff**
There is also an issue on lines 80 to 82, it is not formatting properly on the page itself.  I'm not sure why it is behaving this way.  It should have the green window background and "INFO" but it does not render that way on the help page itself. It should look like how the text on lines 99 - 101 renders. On the help page itself it displays as plain text, ":::important Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts. :::"

## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by bnartiff** [comment]
The article references a Step "B" but as far as I can tell there is no Step B. I think it's referencing Step 1.

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ These configuration instructions apply to log collection from all AWS Source typ
 1. **Log File Discovery.** You have the option to set up Amazon Simple Notification Service (SNS) to notify Sumo Logic of new items in your S3 bucket. A scan interval is required and automatically applied to detect log files.
 
     :::important
-    Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts.
+    Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-   to-date Alerts.
```
Comment (stacykor) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:81]: Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts.

Comment (stacykor) [reply docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:81]: That space doesn't look correct.

Comment (bnartiff) [reply docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:81]: Ahh - it's definitely not. Typo on my end.

**Comment by bnartiff** [comment]
Screen Shot 2023-01-19 at 3 27 11 PM

This is an image of the unformatted plain text on the doc I mentioned above. Not sure it's supposed to display that way.

**Comment by kimsauce** [comment]
> Screen Shot 2023-01-19 at 3 27 11 PM
> 
> This is an image of the unformatted plain text on the doc I mentioned above. Not sure it's supposed to display that way.

Yes you're correct in that it's broken. I assigned this ticket to content owner @JainM6 Monday and will follow up with her again today.

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 2]
```diff
@@ -77,9 +77,9 @@ These configuration instructions apply to log collection from all AWS Source typ
 
 1. **Log File Discovery.** You have the option to set up Amazon Simple Notification Service (SNS) to notify Sumo Logic of new items in your S3 bucket. A scan interval is required and automatically applied to detect log files.
 
-    :::important
+:::important
     Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:81]:
```suggestion
Sumo Logic highly recommends using an SNS Subscription Endpoint for its ability to maintain low-latency collection. This is essential to support up-to-date Alerts.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 3]
```diff
@@ -77,9 +77,9 @@ These configuration instructions apply to log collection from all AWS Source typ
...
-    :::
+:::
 
    * **Scan Interval.** Sumo Logic will periodically scan your S3 bucket for new items in addition to SNS notifications. **Automatic** is recommended to not incur additional AWS charges. This sets the scan interval based on if subscribed to an SNS topic endpoint and how often new files are detected over time. If the Source is not subscribed to an SNS topic and set to **Automatic** the scan interval is 5 minutes. You may enter a set frequency to scan your S3 bucket for new data. To learn more about Scan Interval considerations, see [About setting the S3 Scan Interval](aws-s3-scan-interval-sources.md).
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:84]:
```suggestion
   * **Scan Interval.** Sumo Logic will scan your S3 bucket for new items on a regular basis, in addition to SNS notifications. **Automatic** is recommended to avoid extra AWS charges. This determines the scan interval based on whether you are subscribed to an SNS topic endpoint and how frequently new files are detected over time. The scan interval is 5 minutes if the Source is not subscribed to an SNS topic and is set to **Automatic**.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 4]
```diff
@@ -77,9 +77,9 @@ These configuration instructions apply to log collection from all AWS Source typ
...
    * **SNS Subscription Endpoint** (**Highly Recommended**). New files will be collected by Sumo Logic as soon as the notification is received. This will provide faster collection versus having to wait for the next scan to detect the new file.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:85]: I'd rephrase this as:
```suggestion
   * **SNS Subscription Endpoint** (**Highly Recommended**). Sumo Logic will start collecting new files as soon as it receives a notification. This allows for faster collection instead of having to wait for the next scan to detect the new file.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 5]
```diff
@@ -96,13 +96,13 @@ The following steps use the AWS SNS Console. You may instead use AWS CloudForma
 
 1. Go to **Services \>** **Simple Notification Service** and click **Create Topic**. Enter a **Topic name** and click **Create topic**. Copy the provided **Topic ARN**, you’ll need this for the next step.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:97]:
```suggestion
1. Go to **Services** > **Simple Notification Service** and click **Create Topic**. Enter a **Topic name** and click **Create topic**. Copy the provided **Topic ARN**, you’ll need this for the next step.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 6]
```diff
@@ -96,13 +96,13 @@ The following steps use the AWS SNS Console. You may instead use AWS CloudForma
...
 
-    :::important
-    Make sure that the topic and the bucket are in the same region.
-    :::
+:::important
+Make sure that the topic and the bucket are in the same region.
+:::
 
-1. Again go to **Services \>** **Simple Notification Service** and click **Create Subscription**. Paste the **Topic ARN** from step B above. Select **HTTPS** as the protocol and enter the **Endpoint** URL provided while creating the S3 source in Sumo Logic. Click **Create subscription** and a confirmation request will be sent to Sumo Logic. The request will be automatically confirmed by Sumo Logic.
+1. Again go to **Services \>** **Simple Notification Service** and click **Create Subscription**. Paste the **Topic ARN** from step 1 above. Select **HTTPS** as the protocol and enter the **Endpoint** URL provided while creating the S3 source in Sumo Logic. Click **Create subscription** and a confirmation request will be sent to Sumo Logic. The request will be automatically confirmed by Sumo Logic.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:103]:
```suggestion
1. Again go to **Services** > **Simple Notification Service** and click **Create Subscription**.

Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:103]:
```suggestion
1. Again go to **Services** > **Simple Notification Service** and click **Create Subscription**. Paste the **Topic ARN** from step 1 above. Select **HTTPS** as the protocol and enter the **Endpoint** URL provided while creating the S3 source in Sumo Logic. Click **Create subscription** and a confirmation request will be sent to Sumo Logic. The request will be automatically confirmed by Sumo Logic.
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 7]
```diff
@@ -96,13 +96,13 @@ The following steps use the AWS SNS Console. You may instead use AWS CloudForma
...
 
-1. Select the **Topic** created in step B and navigate to **Actions \> Edit Topic Policy**. Use the following policy template, replace the `SNS-topic-ARN` and `bucket-name` placeholders in the `Resource` section of the JSON policy with your actual SNS topic ARN and S3 bucket name:
+1. Select the **Topic** created in step 1 and navigate to **Actions \> Edit Topic Policy**. Use the following policy template, replace the `SNS-topic-ARN` and `bucket-name` placeholders in the `Resource` section of the JSON policy with your actual SNS topic ARN and S3 bucket name:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:105]:
```suggestion
1. Select the **Topic** created in step 1 and navigate to **Actions** > **Edit Topic Policy**. Use the following policy template, replace the `SNS-topic-ARN` and `bucket-name` placeholders in the `Resource` section of the JSON policy with your actual SNS topic ARN and S3 bucket name:
```

File: docs/send-data/hosted-collectors/amazon-aws/aws-sources.md [hunk 8]
```diff
@@ -125,7 +125,7 @@ The following steps use the AWS SNS Console. You may instead use AWS CloudForma
     }
     ```
 
-1. Go to **Services \> [S3](https://s3.console.aws.amazon.com/s3/buckets/)** and select the bucket to which you want to attach the notifications. Navigate to **Properties \> Events \> Add Notification**. Enter a **Name** for the event notification. In the **Events** section select **All object create events**. In the **Send to** section (notification destination) select **SNS Topic**. An **SNS** section becomes available, select the name of the topic you created in step B from the dropdown. Click **Save**.
+1. Go to **Services \> [S3](https://s3.console.aws.amazon.com/s3/buckets/)** and select the bucket to which you want to attach the notifications. Navigate to **Properties \> Events \> Add Notification**. Enter a **Name** for the event notification. In the **Events** section select **All object create events**. In the **Send to** section (notification destination) select **SNS Topic**. An **SNS** section becomes available, select the name of the topic you created in step 1 from the dropdown. Click **Save**.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/amazon-aws/aws-sources.md:128]:
```suggestion
1. Go to **Services** > **[S3](https://s3.console.aws.amazon.com/s3/buckets/)** and select the bucket to which you want to attach the notifications. Navigate to **Properties** > **Events** > **Add Notification**. Enter a **Name** for the event notification. In the **Events** section, select **All object create events**. In the **Send to** section (notification destination), select **SNS Topic**. When the **SNS** section becomes available, select the name of the topic you created in step 1 from the dropdown. Click **Save**.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Some Formatting changes

**Comment by JainM6** [comment]
> > Screen Shot 2023-01-19 at 3 27 11 PM
> > This is an image of the unformatted plain text on the doc I mentioned above. Not sure it's supposed to display that way.
> 
> Yes you're correct in that it's broken. I assigned this ticket to content owner @JainM6 Monday and will follow up with her again today.
@kimsauce I think we need a correction here. I am not the owner of this doc, this was done before I joined the organization.

**Comment by JainM6** [comment]
> > > Screen Shot 2023-01-19 at 3 27 11 PM
> > > This is an image of the unformatted plain text on the doc I mentioned above. Not sure it's supposed to display that way.
> > 
> > 
> > Yes you're correct in that it's broken. I assigned this ticket to content owner @JainM6 Monday and will follow up with her again today.
> > @kimsauce I think we need a correction here. I am not the owner of this doc, this was done before I joined the organization.

**Comment by kimsauce** [comment]
> > > > Screen Shot 2023-01-19 at 3 27 11 PM
> > > > This is an image of the unformatted plain text on the doc I mentioned above. Not sure it's supposed to display that way.
> > > 
> > > 
> > > Yes you're correct in that it's broken. I assigned this ticket to content owner @JainM6 Monday and will follow up with her again today.
> > > @kimsauce I think we need a correction here. I am not the owner of this doc, this was done before I joined the organization.

@JainM6 , to clarify, each docs writer is responsible for different topics. I'm responsible for observability/traces/OTEL, etc., Marie is CSE/metrics, Nishant does Apps, and `/send-data` docs (in this PR) falls under your purview. The edits I make are almost all from before I joined the org :) cc @stacykor

**Comment by JainM6** [comment]
> > > > > Screen Shot 2023-01-19 at 3 27 11 PM
> > > > > This is an image of the unformatted plain text on the doc I mentioned above. Not sure it's supposed to display that way.
> > > > 
> > > > 
> > > > Yes you're correct in that it's broken. I assigned this ticket to content owner @JainM6 Monday and will follow up with her again today.
> > > > @kimsauce I think we need a correction here. I am not the owner of this doc, this was done before I joined the organization.
> 
> @JainM6 , to clarify, each docs writer is responsible for different topics. I'm responsible for observability/traces/OTEL, etc., Marie is CSE/metrics, Nishant does Apps, and `/send-data` docs (in this PR) falls under your purview. The edits I make are almost all from before I joined the org :) cc @stacykor

@kimsauce I think you have completely misunderstood what I said. You mentioned "I am the content owner of this doc", that's not correct, I did not create this doc. If I were in your place, I'd have simply forwarded this request and informed the requester about the new "Area owner."  That's just my approach. Thanks

**Review by JainM6 — APPROVED** [review]

**Comment by bnartiff** [comment]
I think this is still broken. The info window on the page is displaying in code block format. The same for the text below, "To set up the subscription you need to get an endpoint URL from Sumo to provide to AWS. This process will save your Source and begin scanning your S3 bucket when the endpoint URL is generated. Click on **Create URL** and use the provided endpoint URL when creating your subscription in step C." 

Screenshot 2023-01-20 at 3 16 46 PM

**Comment by kimsauce** [comment]
> I think this is still broken. The info window on the page is displaying in code block format. The same for the text below, "To set up the subscription you need to get an endpoint URL from Sumo to provide to AWS. This process will save your Source and begin scanning your S3 bucket when the endpoint URL is generated. Click on **Create URL** and use the provided endpoint URL when creating your subscription in step C."
> 
> Screenshot 2023-01-20 at 3 16 46 PM

@JainM6 ☝️

**Comment by JainM6** [comment]
@kimsauce @bnartiff  Thank you for pointing this out. I have created a new PR #1433
